flow/layers/transform_layer_unittests.cc

```diff
@@ -100,6 +100,42 @@ TEST_F(TransformLayerTest, Simple) {
                    MockCanvas::DrawCall{1, MockCanvas::RestoreData{0}}}));
 }
 
+TEST_F(TransformLayerTest, Cached) {
+  SkPath child_path;
+  child_path.addRect(5.0f, 6.0f, 20.5f, 21.5f);
+  SkRect cull_rect = SkRect::MakeXYWH(2.0f, 2.0f, 14.0f, 14.0f);
+  SkMatrix initial_transform = SkMatrix::Translate(-0.5f, -0.5f);
```

Comment:
Shall we make a more complicated unit test to make sure all the matrix math is correct? For example, let initial_transform and layer_transform both have offset, scale, and rotation in them?